### PR TITLE
New post: BitTorrent Explained

### PR DESCRIPTION
**Pull request by mahirjain25**
Description: This article gives a basic explanation of the BitTorrent data transfer protocol

### CODE REVIEW

**Comment by salman-bhai** [comment]
Can you add the other articles as well?

**Comment by mahirjain25** [comment]
Is this post ok? Or are there any changes needed?

File: blog_src/_posts/2017-09-02-BitTorrentExplained.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Torrents
+- P2P
+
+categories:
+- CompSoc
+github_username: 'mahirjain25'
+---
+
+
+# BitTorrent Explained
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-02-BitTorrentExplained.md]: I don't think there's a need for this tag as there is already a default heading taken from the file name! Check in local once to see if this is needed!

File: blog_src/_posts/2017-09-02-BitTorrentExplained.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Using BitTorrent is different from any other common download or upload. Usually, users download from a single host server, but this protocol allows users to become a host , joining several other such user-hosts in the "swarm", and upload as well as download files all at the same time. Consider the following simple example:
+
+Several thousand users wish to download a large video file. The user responsible for uploading this file initially, becomes the _seeder_. The users who wish to download this file are known as _leachers_.
+
+Now, each of the leachers begin to download segments/pieces of the file from the seeder. As they do so, they simultaneously begin to upload the pieces they receive to other users in need. Each of these pieces are downloaded non-sequentially, and this is where the client comes in. It **rearranges the pieces** so that they are in order, and **determines what pieces are yet to be recieved.**
+
+Once the file is completely downloaded, the leacher becomes an additional seeder, and the greater the number of such transitions, the better the "health" of the file.
+
+
+## Why is this benefitial?
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-02-BitTorrentExplained.md]: _benefecial_ maybe! :smile:

**Comment by salman-bhai** [comment]
Other articles are there in the IEEE NITK folder that was sent!

File: blog/3D-Printing/index.html [hunk 3]
```diff
@@ -150,9 +150,9 @@ <h1 class="post-title" itemprop="name">3D Printing</h1>
         <section class="post" itemscope itemtype="http://schema.org/BlogPosting">
 
             <article role="article" id="scroll" class="post-content" itemprop="articleBody">
-                <h1 id="3-d-printing-a-world-beyond-2d">3-D Printing-A world beyond 2D</h1>
+                <h1 id="d-printing-a-world-beyond-2d">3-D Printing-A world beyond 2D</h1>
```
Comment (adithyabhatkajake) [inline blog/3D-Printing/index.html]: Why is this happening?What version of jekyll are you using?

**Comment by adithyabhatkajake** [comment]
Closed because the updated version is in #65.

**Pull request closed without merging**